src/System.Management.Automation/utils/PathUtils.cs

```diff
@@ -443,6 +443,10 @@ internal static bool TryDeleteFile(string filepath)
                 {
                     // file is in use on Windows
                 }
+                catch (UnauthorizedAccessException)
+                {
+                    // user does not have permissions
```

Comment:
Make sense to add a comment that AV can block the test and although AppLocker possibly works we are forced to return false?